.dir-locals.el

```diff
@@ -0,0 +1,3 @@
+((c-mode . ((fill-column . 109)
```

Comment:
what's this?

Reply:
to set correctly the formatting mode we use in Emacs :-)

Reply:
just use vim instead :wink: